spec/models/manageiq/providers/vmware/cloud_manager/orchestration_template_dialog_service_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe ManageIQ::Providers::Vmware::CloudManager::OrchestrationTemplateDialogService do
+  let(:template_vapp) { FactoryGirl.create(:orchestration_template_vmware_cloud_with_content) }
+
+  describe "#create_dialog" do
+    it "creates a dialog from VMware vCloud vApp template with stack basic info and parameters" do
+      dialog = subject.create_dialog("test", template_vapp)
+
```

Comment:
Hot and Cfn check some basic dialog [properties](https://github.com/ManageIQ/manageiq/pull/12448/files#diff-bc1e2efabfdc8fecb5a10e0eef9f294eR10). Do you think we should have them here and in Azure spec?